cmake/ev-cli.cmake

```diff
@@ -15,12 +12,26 @@ macro(setup_ev_cli)
         if(NOT ${IS_PYTHON_VENV_ACTIVE})
             message(FATAL_ERROR "Python venv is not active. Please activate the python venv before running this command.")
         endif()
-        set(EV_CLI "${${PROJECT_NAME}_PYTHON_VENV_PATH}/bin/ev-cli")
-        add_dependencies(ev-cli
-            ev-dev-tools_pip_install_dist
+
+        get_target_property(PACKAGE_SOURCE_DIRECTORY ev-dev-tools_pip_install_local PACKAGE_SOURCE_DIRECTORY)
```

Comment:
Hm, this property is only available if [ev_create_pip_install_targets()](https://github.com/EVerest/everest-utils/blob/main/ev-dev-tools/CMakeLists.txt#L1) is called. In this case we don't want to use targets to install ev-cli, so maybe we should have a function in everest-cmake as `ev_create_python_package()` which then creates like <PACKAGE_NAME>_python_package, which only contains the property and in addition the install targets

`ev_pip_install_local()` could then just take `<PACKAGE_NAME>_python_package` as argument without the need of two args

Sorry would have been more helpful to have this idea while reviweing https://github.com/EVerest/everest-cmake/pull/14

Reply:
I just realized that the proposal branches from @a-w50 weren't (draft) PRs yet, so I overlooked them. I think they address this:
https://github.com/EVerest/everest-cmake/pull/15
https://github.com/EVerest/everest-utils/pull/158